src/price.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file Steem price APIs.
+ * @author Johan Nordberg <johan@steemit.com>
+ */
+
+import {JsonRpcError, JsonRpcMethodContext as JCtx} from '@steemit/koa-jsonrpc'
+import {Asset, Price} from 'dsteem'
+
+import {client} from './common'
+
+const ONE_STEEM = new Asset(1, 'STEEM')
+const ORDER_DEPTH = 10 // how many orders to use when calculating the STEEM<>SBD price
```

Comment:
If not weighing by volume, not sure we gain anything by loading more than 1 order on each side. I'd recommend either keeping it dead-simple or going further w/ volume weighing.